build_engine_test_images/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -z "$TF_VAR_build_engine_aws_access_key" ]]; then
+    echo "Must provide TF_VAR_build_engine_aws_access_key in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_build_engine_aws_secret_key" ]]; then
+    echo "Must provide TF_VAR_build_engine_aws_secret_key in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_docker_id" ]]; then
+    echo "Must provide TF_VAR_docker_id in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_docker_password" ]]; then
+    echo "Must provide TF_VAR_docker_password in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_docker_repo" ]]; then
+    echo "Must provide TF_VAR_docker_repo in environment" 1>&2
+    exit 1
+fi
+
+export TF_VAR_tf_workspace=$(pwd)
+
+# Build amd64 images
+export TF_VAR_build_engine_arch="amd64"
+export TF_VAR_build_engine_aws_instance_type="t2.xlarge"
```

Comment:
why we need to export these values?

Reply:
From main.tf `sudo /tmp/generate_images.sh ${var.build_engine_arch} ${var.docker_id} ${var.docker_password} ${var.docker_repo}`